README.md

```diff
@@ -204,6 +204,18 @@ Useful if some specific configuration like --user-principal=host/name@REALM or -
 See man realm for details.
 Example: ad_integration_join_parameters: "--user-principal host/client007@EXAMPLE.COM"
 
+#### ad_integration_sssd_settings
+
+A list of setting to be included into the `[sssd]` section
+of the sssd.conf file. See sssd.conf man pages for details.
+Example:
+
+```yaml
+ad_integration_sssd_settings:
```

Comment:
This should be defined in `defaults/main.yml` like this:

```yaml
# A list of custom setting to be included into the [sssd] section
# of the sssd.conf file. The list will be composed of two entry:
# - key: "configuration_name"
#   value: "configuration_value"
ad_integration_sssd_settings: []
```

That should solve the test failures related to "undefined variable"

Reply:
I have updated the commit